### DISCUSSION TITLE
Should `TextInputMenuAgent` be an unsandboxed process on macOS?

### DISCUSSION THREAD

**Discussion by ghost** [utmapp/UTM#6843] [macOS] [answered]
When I launch a VM (macOS guest) an unsandboxed process called `TextInputMenuAgent` launches as well. Is this process normally unsandboxed? I've downloaded UTM from GitHub and not from the App Store so I was wondering if this process would be sandboxed if I installed the app from the App Store.

**Comment by osy** [comment, marked as answer]
This is part of macOS and not UTM
